src/ucm/util/log.c

```diff
@@ -258,7 +259,7 @@ void __ucm_log(const char *file, unsigned line, const char *function,
     gettimeofday(&tv, NULL);
     ucm_log_snprintf(buf, UCM_LOG_BUG_SIZE - 1, "[%lu.%06lu] [%s:%d] %18s:%-4d UCX  %s ",
                      tv.tv_sec, tv.tv_usec, ucm_log_hostname, getpid(),
-                     basename(file), line, ucm_log_level_names[level]);
+                     ucs_basename(file), line, ucm_log_level_names[level]);
```

Comment:
here was issue: basename operates by mutable string, really don't know how it compiled before